webpack/components/extensions/HostDetails/Tabs/ErrataTab.js

```diff
@@ -117,13 +135,43 @@ export const ErrataTab = () => {
     </Split>
   );
 
+  let toggleGroup;
+  if (!contentFacet.contentViewDefault && !contentFacet.lifecycleEnvironmentLibrary) {
+    toggleGroup = (
+      <ToggleGroup aria-label="Installable Errata">
+        <ToggleGroupItem
+          text={__('All')}
+          buttonId="allToggle"
+          aria-label="Show All"
+          isSelected={!errataInstallable}
+          onChange={() => setErrataInstallable(false)}
+        />
+
+        <ToggleGroupItem
+          text={__('Installable')}
+          buttonId="installableToggle"
+          aria-label="Show Installable"
+          isSelected={errataInstallable}
+          onChange={() => setErrataInstallable(true)}
+        />
+      </ToggleGroup>
+    );
+  }
+
   return (
     <div>
       <div id="errata-alert">
         <Alert variant="info" isInline title={__('Note')}>
           {__('Errata management functionality on this page is incomplete.')} {' '}
           <a href={urlBuilder(`content_hosts/${hostId}/errata`, '')}>{__('Visit the previous Errata page') }.</a>
         </Alert>
+        { !errataInstallable &&
+        <Alert variant="info" isInline title={__('Note')}>
+          {__(" Some of the Errata shown below may not be installable as they are not in this Host's Content View and Lifecycle Environment.")}
+          {__('In order to apply such Errata an Incremental Update is required. ')}
+          <a href={urlBuilder('errata', '')}>{__('Select Errata for an Incremental Update.') }.</a>
+        </Alert>
+         }
```

Comment:
Well I want to make sure the message is clear when you select `All` it will include errata that you cant choose to apply

Reply:
Actually it takes you to All errata across every repo page. May not be as useful

Reply:
Suggestion for tooltip text
```
This erratum is not installable because the host does not have access to it through its content view and lifecycle environment.
```